### PR TITLE
ticdc: update interruption FAQs

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

* ticdc: aggregate interruption FAQs
* update oom faq 
* ticdc: add partition table faq, close https://github.com/pingcap/ticdc/issues/2404

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -42,35 +101,38 @@ aliases: ['/docs-cn/dev/ticdc/troubleshoot-ticdc/','/docs-cn/dev/reference/tools
...
-- 通过 Grafana 检查 `exit error count` 监控项，该监控项大于 0 代表同步任务出现错误。
-- 通过 `cdc cli changefeed list` 和 `cdc cli changefeed query` 命令查看同步任务的状态信息。任务状态为 `stopped` 代表同步中断，`error` 项会包含具体的错误信息。任务出错后可以在 TiCDC server 日志中搜索 `error on running processor` 查看错误堆栈，帮助进一步排查问题。
-- 部分极端异常情况下 TiCDC 出现服务重启，可以在 TiCDC server 日志中搜索 `FATAL` 级别的日志排查问题。
+- 在 v4.0.13 及以下版本中 TiCDC 同步分区表存在问题，导致同步停止。
 
-## TiCDC 的 `gc-ttl` 是什么？
-
-从 TiDB v4.0.0-rc.1 版本起，PD 支持外部服务设置服务级别 GC safepoint。任何一个服务可以注册更新自己服务的 GC safepoint。PD 会保证任何晚于该 GC safepoint 的 KV 数据不会在 TiKV 中被 GC 清理掉。在 TiCDC 中启用了这一功能，用来保证 TiCDC 在不可用、或同步任务中断情况下，可以在 TiKV 内保留 TiCDC 需要消费的数据不被 GC 清理掉。
+    - 该场景下 TiCDC 会保存任务信息，由于 TiCDC 已经在 PD 中设置的 service GC safepoint，在 `gc-ttl` 的有效期内，同步任务 checkpoint 之后的数据不会被 TiKV GC 清理掉。
+    - 处理方法：
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:107]:
```suggestion
    - 处理方法：

```

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -42,35 +101,38 @@ aliases: ['/docs-cn/dev/ticdc/troubleshoot-ticdc/','/docs-cn/dev/reference/tools
...
+        1. 通过 `cdc cli changefeed pause -c <changefeed-id>` 暂停同步。
+        2. 等待约一分钟后，通过 `cdc cli changefeed resume -c <changefeed-id>` 恢复同步。
 
-启动 TiCDC server 时可以通过 `gc-ttl` 指定 GC safepoint 的 TTL，这个值的含义是当 TiCDC 服务全部挂掉后，由 TiCDC 在 PD 所设置的 GC safepoint 保存的最长时间，该值默认为 86400 秒。
+### 同步任务中断，尝试再次启动后 TiCDC 发生 OOM，如何处理？
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:111]:
```suggestion
### 同步任务中断，尝试再次启动后 TiCDC 发生 OOM，应该如何处理？
```

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -42,35 +101,38 @@ aliases: ['/docs-cn/dev/ticdc/troubleshoot-ticdc/','/docs-cn/dev/reference/tools
...
 
-## 同步任务中断，尝试再次启动后 TiCDC 发生 OOM，如何处理？
+升级 TiDB 集群和 TiCDC 集群到最新版本。OOM 问题在 v4.0.14 上已得到缓解。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:113]:
```suggestion
升级 TiDB 集群和 TiCDC 集群到最新版本。该 OOM 问题在 v4.0.14 上已得到缓解。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: ticdc/troubleshoot-ticdc.md [hunk 4]
```diff
@@ -42,35 +101,38 @@ aliases: ['/docs-cn/dev/ticdc/troubleshoot-ticdc/','/docs-cn/dev/reference/tools
...
+升级 TiDB 集群和 TiCDC 集群到最新版本。该 OOM 问题在 v4.0.14 上已得到缓解。
 
-如果同步任务长时间中断，累积未消费的数据比较多，再次启动 TiCDC 可能会发生 OOM。这种情况下可以启用 TiCDC 提供的实验特性 Unified Sorter 排序引擎，该功能会在系统内存不足时使用磁盘进行排序。启用的方式是创建同步任务时在 `cdc cli` 内传入 `--sort-engine=unified` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下：
+如无法升级，则需要开启 Unified Sorter 排序功能，该功能会在系统内存不足时使用磁盘进行排序。启用的方式是创建同步任务时在 `cdc cli` 内传入 `--sort-engine=unified` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-cdc cli changefeed update -c [changefeed-id] --sort-engine="unified" --sort-dir="/data/cdc/sort" --pd=http://10.0.10.25:2379
+cdc cli changefeed update -c <changefeed-id> --sort-engine="unified" --sort-dir="/data/cdc/sort" --pd=http://10.0.10.25:2379
```
Comment (liuzix) [inline ticdc/troubleshoot-ticdc.md:120]: `--sort-dir` is no longer supported now

**Review by liuzix — APPROVED** [review]

File: ticdc/troubleshoot-ticdc.md [hunk 5]
```diff
@@ -42,35 +101,46 @@ aliases: ['/docs-cn/dev/ticdc/troubleshoot-ticdc/','/docs-cn/dev/reference/tools
...
+
+{{< copyable "shell-regular" >}}
 
-## 同步任务中断，尝试再次启动后 TiCDC 发生 OOM，如何处理？
+```shell
+cdc cli changefeed update -c <changefeed-id> --sort-engine="unified" --pd=http://10.0.10.25:2379
+```
 
-如果同步任务长时间中断，累积未消费的数据比较多，再次启动 TiCDC 可能会发生 OOM。这种情况下可以启用 TiCDC 提供的实验特性 Unified Sorter 排序引擎，该功能会在系统内存不足时使用磁盘进行排序。启用的方式是创建同步任务时在 `cdc cli` 内传入 `--sort-engine=unified` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下：
+如无法升级，对于**之前的版本**，启用的方式是创建同步任务时在 `cdc cli` 内传入 `--sort-engine=unified` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下：
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:123]:
```suggestion
如果无法升级到上述版本，需要在**之前的版本**上开启 Unified Sorter，可以在创建同步任务时在 `cdc cli` 内传入 `--sort-engine=unified` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下：
```

**Pull request merged**